tests/integration/remote/invoke/test_remote_invoke.py

```diff
@@ -0,0 +1,284 @@
+import json
+import uuid
+import base64
+
+from parameterized import parameterized
+
+from tests.integration.remote.invoke.remote_invoke_integ_base import RemoteInvokeIntegBase
+from tests.testing_utils import run_command
+
+from pathlib import Path
+import pytest
+
+
+@pytest.mark.xdist_group(name="sam_remote_invoke_single_lambda_resource")
```

Comment:
what is the value of this mark, are we going to run the remote invoke test cases in parallel?

Reply:
Any new tests added into "OtherTests" group will run the tests in parallel, by grouping them with the `xdist_group` above. With this added, pytest will make sure that the tests will be executed in the same group so that we can deploy -> run tests -> delete stack.

If this wasn't provided, the tests will be split into different workers, causing re-deployment of this stack and eventually will took longer time than it should.